### PR TITLE
Add spacing and change some fonts

### LINKED ISSUE #266: QA: People Page - IMPORTANT
- [x] Add quote behind characters and quote seems to be cut on top (see red ellipse) **IMPORTANT**
quote

- [x] Increase space between name and role (quote side). Maybe add the class "mt-2"?
screen shot 2017-04-05 at 5 47 43 pm

- [x] Small gray: line height 1.5  - _I know it affects all the pages too and all it seems fine to me. Let me know if you find some weird case if that change happens._
screen shot 2017-04-05 at 5 10 21 pm
 
- [x] Add a bit more space between top or each card and first line of text (see red rectangles). It seems too tight.
cards

- [x]  in the quotes, let's use “Nunito Sans Light Italic” (instead of using “font-style:italic”)
italics

- [x] Add grey slant

[svg here](https://drive.google.com/open?id=0B3b1KWRwjxtoNWVYbXowajZ3eTQ)
screen shot 2017-04-05 at 5 26 34 pm

- [x] Reduce size of "more details" button (we don't want it to compete with main CTA). Changing the font-size and padding seems to do the trick.

screen shot 2017-04-05 at 4 55 27 pm

### PR DESCRIPTION
**Pull request by alanmoo**
Closes #266

### CODE REVIEW

File: source/sass/utilities.scss [hunk 1]
```diff
@@ -10,6 +10,10 @@
   background-image: url(../_images/slanting-lines-pttn.svg);
   background-size: 150px;
   width: 100%;
+
+  &.gray {
```
Comment (gvn) [inline source/sass/utilities.scss]: Can we make this `slanty-background-gray` since it's a variant?

Comment (alanmoo) [inline source/sass/utilities.scss]: So the element uses `class= "slanty-background slanty-background-gray"`?

**Comment by alanmoo** [comment]
Double check this is what you wanted?

**Comment by gvn** [comment]
Yep LGTM

**Pull request merged**